### DISCUSSION TITLE
Unable to verify as a student, keeps rejecting.

### DISCUSSION THREAD

**Discussion by PiyushDuggal-source** [community/community#51961] [GitHub Education]
Hey, I need some help please.
I am having similar problem, I was not aware of setting the full name but still after setting the full name I am getting rejected. I have used my latest ID card first time and my admission letter second time, still it did not verify me.
Now I am unable to use any of these as its showing "Your uploaded proof has already been used previously.".
Please let me know what should I do.
Thanks..

**Comment by Ntshangase** [comment]
Hi,

maybe you should try using your Proof of registration which shows that you are registered for this year. Also make sure that you have added your school email to your GitHub account.

**Comment by samus-aran** [comment]
Hi @PiyushDuggal-source 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Hope this helps! Let us know how you get along 👍
